curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,527 @@ Which of the following is the default value of the `position` property?
+
+---
+
+It positions the element relative to its nearest ancestor.
+
+#### --answer--
+
+It positions the element relative to the viewport and keeps it fixed during scrolling.
+
+### --question--
+
+#### --text--
+
+Which value of the `position` property allows an element to stick to a defined position when scrolling past a certain point?
+
+#### --distractors--
+
+`absolute`
+
+---
+
+`relative`
+
+---
+
+`fixed`
+
+#### --answer--
+
+`sticky`
+
+### --question--
+
+#### --text--
+
+What is the purpose of the `z-index` property in CSS?
+
+#### --distractors--
+
+To control the horizontal alignment of elements.
+
+---
+
+To define the visibility of elements.
+
+---
+
+To set the background color of elements.
+
+#### --answer--
+
+To control the vertical stacking order of overlapping elements.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `position: relative`?
+
+#### --distractors--
+
+```css
+.box {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+}
+```
+
+---
+
+```css
+.box {
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+```
+
+---
+
+```css
+.box {
+  position: static;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: relative;
+  top: 10px;
+  left: 10px;
+}
+```
+
+### --question--
+
+#### --text--
+
+What happens to an element with `position: static`?
+
+#### --distractors--
+
+It is removed from the normal document flow.
+
+---
+
+It overlaps other elements.
+
+---
+
+It sticks to the viewport during scrolling.
+
+#### --answer--
+
+It follows the normal document flow.
+
+### --question--
+
+#### --text--
+
+Which of the following is true about `position: absolute`?
+
+#### --distractors--
+
+It positions the element relative to the viewport.
+
+---
+
+It keeps the element in the normal document flow.
+
+---
+
+It positions the element relative to the document root.
+
+#### --answer--
+
+It positions the element relative to its nearest positioned ancestor.
+
+### --question--
+
+#### --text--
+
+What is the difference between `position: fixed` and `position: sticky`?
+
+#### --distractors--
+
+`fixed` positions the element relative to its parent, while `sticky` positions it relative to the viewport.
+
+---
+
+`fixed` keeps the element in the normal flow, while `sticky` removes it.
+
+---
+
+`fixed` allows scrolling, while `sticky` does not.
+
+#### --answer--
+
+`fixed` positions the element relative to the viewport, while `sticky` positions it relative to its nearest scrollable ancestor.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `z-index`?
+
+#### --distractors--
+
+```css
+.box {
+  position: static;
+  z-index: 10;
+}
+```
+
+---
+
+```css
+.box {
+  position: relative;
+}
+```
+
+---
+
+```css
+.box {
+  position: absolute;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: absolute;
+  z-index: 5;
+}
+```
+
+### --question--
+
+#### --text--
+
+What is the default stacking context in CSS?
```

Comment:
not sure this was explicitly covered in the lectures or workshop.
so it would be best to come up with different question